htdocs/recruitment/recruitmentjobposition_card.php

```diff
@@ -446,7 +446,7 @@
 			// Close as recruited/canceled
 			if ($object->status == $object::STATUS_VALIDATED) {
 				if ($usercanclose) {
-					print '<a class="butAction" href="'.$_SERVER["PHP_SELF"].'?id='.$object->id.'&amp;action=closeas'.(empty($conf->global->MAIN_JUMP_TAG) ? '' : '#close').'"';
+					print '<a class="butAction" href="'.$_SERVER["PHP_SELF"].'?id='.$object->id.'&action=closeas&token='.newToken().(empty($conf->global->MAIN_JUMP_TAG) ? '' : '// close').'"';
```

Comment:
@ptibogxiv #close is not a comment, i think you dont have to change it

Reply:
@frederic34 i made any change perhaps it was done by stickler-ci....
https://github.com/Dolibarr/dolibarr/pull/22183/commits/4f45c89ecb53252e3e29f51461546193769c2028

Reply:
ah yes it stickler bot